components/support/test/src/test/java/mozilla/components/support/test/robolectric/DelegatesTest.kt

```diff
@@ -0,0 +1,24 @@
+package mozilla.components.support.test.robolectric
+
+import androidx.test.core.app.ApplicationProvider
+import org.junit.Assert
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class DelegatesTest {
+
+    private val context by providedAppContext()
```

Comment:
I was thinking about something similar some time ago. My idea was to add a `context` extension property to `Any` so that it can be used in tests without declaring it every time. What do you think?

Reply:
Sounds interesting and I like the idea but I'm affraid that implicit `val Any.context` can be misleading sometimes. There are cases when we need/want to mock methods in `Context` (e.g. mocking resources manager). But we can make it more explicit with `fun Any.context()`. Method seems like explicit way to say "provide me context" and it will not intersect with _possible_ local `context` variable.

Reply:
Yeah, we can even make it more descriptive and name it `testContext` or similar.

Reply:
Awesome! I like it. :+1:

Reply:
Done.